app/routes/account.$accountId._index/route.tsx

```diff
@@ -0,0 +1,17 @@
+import { Center } from "@mantine/core"
```

Comment:
I think for right now this is the same page intent as `account.accountId.apps` not sure we need both.